saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/ClassifierUtils.scala

```diff
@@ -165,6 +167,14 @@ object ClassifierUtils extends Logging {
     }
   }
 
+  object InitializeClassifiers {
+    def apply[HEAD <: AnyRef](node: Node[HEAD], cl: ConstrainedClassifier[_, HEAD]*)(implicit d1: DummyImplicit, d2: DummyImplicit, d3: DummyImplicit) = {
```

Comment:
This shouldn't need DummyImplicit arguments as this is the only `apply` method in this class.